### ISSUE TITLE
Marauder ship descriptions

### ISSUE THREAD

**Issue by lheckemann** [endless-sky/endless-sky#2433]
The Marauder ships have lovely descriptions! But I don't know of anywhere in-game that I can actually see them — are they actually shown anywhere other than in the shipyard screen? If not, that means that a lot of the (rather enjoyable) descriptions are actually inaccessible.

**Comment by Bladewood** [comment]
well, we have world-forge and the all-content-plugin...

**Comment by lheckemann** [comment]
But they're not accessible in vanilla gameplay, are they?

**Comment by Bladewood** [comment]
meh

**Comment by KiLEdEnNis** [comment]
can't ya just cap a Marauder shjip & read the description?

**Comment by lheckemann** [comment]
Read the description where? :)

**Comment by KiLEdEnNis** [comment]
I'll just retaliate w/ the World Ships having descriptions

just look in the files though

**Comment by lheckemann** [comment]
Yes, I know they do all have descriptions. The issue is that they can't actually be seen anywhere in game, which is a shame because they're good reads.

**Comment by EndrosG** [comment]
Do you have any suggestion how your issue could be addressed?
I mean you know that the descriptions are shown in the shipyard - if you click on one, i.e. if you have one.
And you probably know too, that these descriptions are currently *only* used in the shipyard. Or have you ever seen the (exakt) description of the Star Barge or Fury / ... anywhere outside of the shipyard?

Just asking if you have an idea how to change that.

**Comment by MessyMix** [comment]
Make them like outfits. If you have one, they show up in the shipyard as "owned: X" where X is the number of ships of that kind you own. When you sell it, it becomes an "in stock: Y", just like with outfitters. That way, you can see the descriptions of all the ships you have.

**Comment by lheckemann** [comment]
Exactly, they're only shown in the shipyard where a number of ships can never be found in the vanilla game. I don't have a wonderful idea for how to solve this, but potentially there could be a place for it in the ship info screen.

**Comment by Tadrix** [comment]
@Bladewood All content plugin lets you see basic versions of Marauder ships, not engine+ or weapon+ versions.

**Comment by EndrosG** [comment]
@MessyMix : I like that idea. Because I like a spontanous thought which relates to this.
But before I would start conding anything like this, I would first want to have "enough" feedback (support) for the idea.

The shipyard currently does not offer you to re-buy a ship (hull) after you bought it.
If we start to add the owned but "not sold here" ships to the left catalogue display, we are pretty close to the look and feel of the outfitter and a player might expect the game to offer any ship (hull) to be bought back again if the player accidently sold the wrong hull. - Because the outfitter offers that buy back option.
Now, the downside / abuse potential of this idea is very straight forward.
Asume, you have captured an alien ship with some cool outfits.
You sell the ship. You buy the outfits and install them to your "normal" ships.
You buy the ship (hull) back again from the yard, with the default configuration of that type. You can another set of the cool outfits since the shipyard only sells fully equipped ships.

That effect happens with vanilla ships too. You can buy Syndicate Freighter (150 tons cargo) in Caph and Diphda systems, north from Sol. These systems dont sell Scram Drives regularly. But you can get the Scram Drive outfit there if you buy a freighter and sell it's scram drive and put it on the ship of your choice. -- Just one of many examples which give reason why the shipyard can't be restricted to outfits being locally available. Second example are planets with shipyards but no outfitters.

**Next problem: the savegame does not contain information which variant of the ship it is.
Different variants have different descriptions.**
So if you happen to capture a `Marauder Arrow (Engines)`, your savegame will only contain the information that it is a ship of the `Marauder Arrow` base class. Plus all base attributes etc., but the savegame does not point to the description of the (Engines) version.

**Comment by KiLEdEnNis** [comment]
why not just make a small area in the I screen where the description is shown? maybe something that can be expanded

**Comment by EndrosG** [comment]
> the savegame does not contain information which variant of the ship it is. Different variants have different descriptions.

This thing (imho) needs to be addressed first before creating any information display. Displaying the vanilla Marauder Arrow description does not fit to the Engines variant.

**Comment by MessyMix** [comment]
@EndrosG it was just a thought, and a solution that makes sense. I think you raise awesome points about what needs addressing and all. :D

**Comment by MCOfficer** [comment]
i like the shipyard idea a lot, but there's one thing you need to take into consideration: the interference  between shipyard and outfitter.
now, i'm Bad at explaining, so i'll give you an example: you cap a nice korath Raider. you sell it, and buy it's "in stock" jump drive from the outfitter. now, the "in stock" Raider needs to be either sold _without jump drive_ (as the JD is already gone) or not be sold at all (so that only it's parts are available).
think about this, it might come out quite tricky.

**Comment by EndrosG** [comment]
@MCOfficer 

> Now, the downside / abuse potential of this idea is very straight forward.
> Asume, you have captured an alien ship with some cool outfits.
> You sell the ship. You buy the outfits and install them to your "normal" ships.
...

;-)

**Comment by Nechochwen-D** [comment]
What about just adding a "ship info" outfit to the marauder ships under the special category that has as its description what would normally be in the shipyard?

**Comment by Bladewood** [comment]
well then that raises the question of why is it that only marauder ships have this outfit? why not all of the ships in the game?

**Comment by Nechochwen-D** [comment]
Why not?

**Comment by EndrosG** [comment]
Selling a bare hull only, comes with another issue.
If the planet does not have an outfitter, like the planets of the Wanderer initially have according to the data file `map.txt` [which gets changed by event, though], and you buy only a mere hull, you have no chance to put outfits onto it. And I never succeeded in bypassing the preflight checks / exit from the preflight checks to sell the ship instead of putting outfits onto it. - You would say that in this case the player also has no chance to buy back the cool alien outfits before buying the hull again. True.
But what I currently see is, that the game will need to differentiate between "this ship is sold here regularly - and thus with all outfits included" and "this ship is not sold here - and thus only with the available outfits". --- And then we still have to hope that no darn modder like me uses an inline ship definition, just because it works and was used for the rand asteroid.

**Comment by Wrzlprnft** [comment]
i'd add  description display to the ship info screen, or the map.

**Comment by EndrosG** [comment]
If we would take the shipyard approach, making these ships visible in the main catalogue on the left side, we *should* also solve issue #2363 (bug: ship prices for variants). If we start displaying that sort of information, we should display the correct ones. ;-)

**Comment by tehhowch** [comment]
With #3070 any ship which can be targeted shows its description (where the planet description would be if you were to click it)
